parents/integrationtest/pom.xml

```diff
@@ -316,51 +300,17 @@
             <id>elasticsearch-5.6</id>
             <properties>
                 <test.elasticsearch.connection.version>${version.org.elasticsearch.latest-5.6}</test.elasticsearch.connection.version>
-                <!-- flavours are only supported starting with ES 6.3 -->
-                <test.elasticsearch.run.flavour></test.elasticsearch.run.flavour>
                 <test.elasticsearch.testdialect>org.hibernate.search.util.impl.integrationtest.backend.elasticsearch.dialect.Elasticsearch5TestDialect</test.elasticsearch.testdialect>
             </properties>
-            <build>
-                <pluginManagement>
-                    <plugins>
-                        <plugin>
-                            <groupId>com.github.alexcojocaru</groupId>
-                            <artifactId>elasticsearch-maven-plugin</artifactId>
-                            <version>${version.com.github.alexcojocaru.elasticsearch.plugin}</version>
-                            <configuration>
-                                <pathConf>${project.build.directory}/elasticsearch-maven-plugin/5.0/configuration/</pathConf>
-                                <pathInitScript>${project.build.directory}/elasticsearch-maven-plugin/5.0/init/init.script</pathInitScript>
-                            </configuration>
-                        </plugin>
-                    </plugins>
-                </pluginManagement>
-            </build>
         </profile>
 
         <!-- Elasticsearch 6.0 to 6.2 test environment -->
         <profile>
             <id>elasticsearch-6.0</id>
             <properties>
                 <test.elasticsearch.connection.version>${version.org.elasticsearch.latest-6.2}</test.elasticsearch.connection.version>
-                <!-- flavours are only supported starting with ES 6.3 -->
-                <test.elasticsearch.run.flavour></test.elasticsearch.run.flavour>
                 <test.elasticsearch.testdialect>org.hibernate.search.util.impl.integrationtest.backend.elasticsearch.dialect.Elasticsearch60TestDialect</test.elasticsearch.testdialect>
             </properties>
-            <build>
-                <pluginManagement>
-                    <plugins>
-                        <plugin>
-                            <groupId>com.github.alexcojocaru</groupId>
-                            <artifactId>elasticsearch-maven-plugin</artifactId>
-                            <version>${version.com.github.alexcojocaru.elasticsearch.plugin}</version>
-                            <configuration>
-                                <pathConf>${project.build.directory}/elasticsearch-maven-plugin/6.0/configuration/</pathConf>
-                                <pathInitScript>${project.build.directory}/elasticsearch-maven-plugin/6.0/init/init.script</pathInitScript>
```

Comment:
So we're no longer creating index templates on startup... I guess that's fine, as it makes the build simpler, and we don't really need the slowlog anyway: we should use performance tests for that.

The templates were also used to reduce the number of replicas, though. That's important in older version (such as 5.6) which default to 5 replicas, which doesn't make sense in a single-node setup. I'll start a full build to see if everything still works even with these default settings.